.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+before_install:
+- npm install -g npm@latest
```

Comment:
This might better illustrate the need for a special npm for iojs:

https://github.com/aredridel/homebrew-iojs/blob/master/Formula/iojs.rb